### PR TITLE
Activity Panel: set reviews unread indicator based on real data

### LINKED ISSUE #1819: Activity Panel: Hook up Reviews Panel Unread to Real Data
Related #1817

See the above issue for more context / possible approach to implement a  REST endpoint to help with this. Currently the Reviews Activity Panel `unread` indicator is [hard-coded](https://github.com/woocommerce/woocommerce-admin/blob/master/client/header/activity-panel/index.js#L117) to `true`.  This should be updated to only be true if pending reviews exist.

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes #1819.

Uses real data in order to show or hide the unread indicator next to the _Reviews_ Activity Panel tab.

### Screenshots
reviews3

### Detailed test instructions:
- Create a product review.
- Go to any `wc-admin` page and verify an unread indicator (red dot) appears next to the _Reviews_ tab of the Activity Panel.
- Click on the tab and verify the new review is also marked with a red dot.
- Close the panel and verify the red dot disappeared from the tab. Click on it again and verify the review doesn't have the red dot anymore either.

### CODE REVIEW

File: client/header/activity-panel/index.js
Comment (psealock) [inline client/header/activity-panel/index.js]: Maybe this second check should be `reviews[ 0 ]. date_created_gmt` because `new Date( undefined + 'Z')` is "Invalid Date".

**Review by psealock — APPROVED** [review]
LGTM @Aljullu,

I see the unread indicator appear on comments that are pre-approved or once they are approved, but not ones that are awaiting approval. It seems to be a good thing to tackle in a follow up PR.

**Review by joshuatf — APPROVED** [review]
Nice- testing well and LGTM!

Noted the same issue @psealock did with unapproved reviews, which we'll definitely want to follow up on since those are probably more "actionable."

On a side note, are there plans for adding content here when no reviews exist?  Maybe a small message like `You have no product reviews yet.`?

File: client/header/activity-panel/index.js
Comment (Aljullu) [reply client/header/activity-panel/index.js]: Fixed in: 11bc68aa3e5f18f2d54aa37fbabe56f54f02278c.

**Comment by Aljullu** [comment]
Thanks for the reviews @psealock and @joshuatf:

> I see the unread indicator appear on comments that are pre-approved or once they are approved, but not ones that are awaiting approval. It seems to be a good thing to tackle in a follow up PR.

> Noted the same issue @psealock did with unapproved reviews, which we'll definitely want to follow up on since those are probably more "actionable."

Created #1868 and will work on it today.

> On a side note, are there plans for adding content here when no reviews exist? Maybe a small message like `You have no product reviews` yet.?

Created an issue for that too: #1869.

**Pull request merged**
